stories/components/slr_carousel/items.js

```diff
@@ -0,0 +1,15 @@
+export default [
+    {
+      link: 'https://earth.gov/sealevel',
+      title: 'Global Sea Level Rise',
+      description: 'The Global Sea Level Rise Portal is an international collaboration that offers essential information on current and project sea level rise to inform coastal communities across the globe.',
+      image: new URL('../media/global_slr.png', import.meta.url).href,
+      imageAlt: 'Coastline with crashing waves'
+    },
+    {
+    link: 'https://earth.gov/sealevel/us',
```

Comment:
Need trailing slash here, too

```suggestion
    link: 'https://earth.gov/sealevel/us/',
```